content/en-us/workspace/streaming.md

```diff
@@ -378,31 +378,31 @@ In some cases, it's necessary to detect when an object streams in or out and rea
 
    ```lua title='LocalScript - CollectionService Streaming Detection' highlight='10-15'
    local CollectionService = game:GetService("CollectionService")
-   local RunService = game:GetService("RunService")
 
-   local random = Random.new(tick())
-   local heartbeatCount = 0
+   local tagName = "FlickerLightSource"
+   local random = Random.new(os.clock())
```

Comment:
```suggestion
   local random = Random.new()
```

Random will pull a seed from a source of entropy when the argument is not specified. Since the goal isn't to produce deterministic randomness, docs (and developers) should avoid passing the seed argument.